frog/imports/ui/GraphEditor/SidePanel/ListComponent.jsx

```diff
@@ -0,0 +1,58 @@
+// @flow
+
+import React, { Component } from 'react';
+
+class ListComponent extends Component {
+  state: { over: boolean };
+  constructor(props: any) {
+    super(props);
+    this.state = {
+      over: false
+    };
+  }
+
+  render() {
+    const obj = this.props.object;
+    const inp = obj.meta.inputs === '' ? '' : 'Inputs : ' + obj.meta.inputs;
+    const InOut = () => (inp !== '' && outp !== '' ? <br /> : <div />);
+    const outp = obj.meta.outputs === '' ? '' : 'Outputs : ' + obj.meta.outputs;
+
+    const onOver = () => this.setState({ over: true });
+    const onOut = () => this.setState({ over: false });
+
+    return (
+      <div value="titi" className="list-group-item">
+        <h5 style={{ fontWeight: 'bold' }}>
+          {' '}{obj.meta.name}{' '}
```

Comment:
That would be prettier (which is triggered by eslint). I have prettier run on each save in my editor, so I see stuff like this right away.